### ISSUE TITLE
Sync Library

### ISSUE THREAD

**Issue by vinhui** [SIMKL/script.simkl#66]
Has there been any progress on the `Sync Library` option?
It seems that the `fetchlibrary` branch has been stale for years now.

**Comment by masyk** [comment]
No, that branch is not working, if anyone can help with this, we can use something new.

**Comment by alsdfjldafsjalsfjk** [comment]
Is this still non-functional? Or did i configure something wrong, i see now option to sync library

**Comment by masyk** [comment]
There is no code in the app to sync full library to\from kodi.

**Comment by alsdfjldafsjalsfjk** [comment]
thats unfortunate, there is a option there.. its just greyed out.
